### DISCUSSION TITLE
DietPi-Software | Organizr

### DISCUSSION THREAD

**Discussion by dragonandy** [MichaIng/DietPi#4986] [Software Request]
#### Formal software information
- Software title |  Organizr 
- Short description | HTPC/Homelab Services Organizer - Written in PHP 
- Official URL (if available) | https://organizr.app/ 
- GitHub URL (if available) | https://github.com/causefx/Organizr
- (Official) install documentation | https://docs.organizr.app/installation/installing-organizr#ubuntu-and-debian

#### Are there similar/alternative software titles available with DietPi-Software?

- HTPC Manager

#### What makes your requested software better than the above solutions, if available?

- opensource software, free to use personally. it can easily integrate with most service managed by dietpi, act like a 'portal' of all home service.

#### How can DietPi make the installation easier or compatible, than following the install instructions or do APT installation, if available?

- Manual step to install is tedious. you need to config php and httpserver, and clone the repo, setup with the right permission and web-data user and more. DietPi automate the installation steps to save a lot of time for less experienced Linux users.

#### Can you provide the installation steps that you would suggest DietPi-Software to do?

1. install webserver stack and developer stack git sqlite php-module etc.
2. config PHP fpm and http server
3. config ownership of the web-root folder
4. initial config organizr (eg. admin user, sqlite db file, keys)
5. integrate with installed services

#### Are you willing to help maintaining the software installation, e.g. in case of needed setup changes due to updates etc.? This is not needed, but could speed up our decision to implement it, as man power is always a topic :wink:.

- yes if I can help. but I am not a professional programmer.

**Comment by MichaIng** [comment]
@dragonandy 
Many thanks for your request. I moved it to discussions where we start to collect software requests now. Users can vote here, which is one benefit, since FeatHub is sadly down, respectively unmaintend and cannot be used anymore.
